### PR TITLE
attr: Minor simplification

### PR DESCRIPTION
**Pull request by TimothyGu**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -116,15 +116,13 @@ function jade_style(val) {
...
-  if ('boolean' == typeof val || null == val) {
-    if (val) {
-      return ' ' + (terse ? key : key + '="' + key + '"');
-    } else {
-      return '';
-    }
+  if (!val && (key === 'class' || key === 'style' || typeof val !== 'number' && typeof val !== 'string')) {
+    return '';
+  }
+  if ('boolean' == typeof val) {
```
Comment (alubbe) [inline index.js]: Wouldn't it be easier to read if it said `if (val === true)`?

File: index.js [hunk 2]
```diff
@@ -116,15 +116,13 @@ function jade_style(val) {
...
 exports.attr = jade_attr;
 function jade_attr(key, val, escaped, terse) {
-  if ((key === 'class' || key === 'style') && !val) return '';
-  if ('boolean' == typeof val || null == val) {
-    if (val) {
-      return ' ' + (terse ? key : key + '="' + key + '"');
-    } else {
-      return '';
-    }
+  if (!val && (key === 'class' || key === 'style' || typeof val !== 'number' && typeof val !== 'string')) {
```
Comment (alubbe) [inline index.js]: It took me a while to get this. How about being more explicit with

``` js
if (val === false || val == null || (!val && (key === 'class' || key === 'style'))
```

**Comment by TimothyGu** [comment]
Thanks for the suggestions. Both of them sound good.

**Comment by ForbesLindesay** [comment]
Awesome

**Pull request merged**